### PR TITLE
Added and updated strings for Update 16

### PR DESCRIPTION
**Pull request by eliasball**

### CODE REVIEW

**Review by PythooonUser — APPROVED** [review]
Looks fine to me.

File: language-pack_de/modInfo.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{workshopId:892615445,title:Deutsche Übersetzung / German translation,image:modPreview.png}
```
Comment (joshuaskelly) [inline language-pack_de/modInfo.json]: Probably shouldn't include this file in the repo. 😉

**Review by joshuaskelly — CHANGES_REQUESTED** [review]

File: language-pack_de/modInfo.json (hunk 1 above)
Comment (eliasball) [reply language-pack_de/modInfo.json]: Removed the file, thanks you Joshua ;D

**Review by joshuaskelly — APPROVED** [review]
Looks good! Sorry for taking so long on this. 🙃

**Comment by eliasball** [comment]
> Looks good! Sorry for taking so long on this. 🙃

No problem mate :D

**Pull request merged**